### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by kthhrv**
readme should exist

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# envars
+
+Command-line Python application to manage environment variables and AWS KMS encrypted secrets in a single file.
+
+## Requirements
+
+Your AWS account should have a KMS key created with an alias of `alias/envars`.
```
Comment (CeeEffEff) [inline README.md:7]: Can this be explained further or a link to a guide provided please? I need an ELI5 version haha

Comment (kthhrv) [reply README.md:7]: link added

**Review by CeeEffEff — APPROVED** [review]
LGTM
